packages/clay-charts/src/__tests__/AreaSplineChart.js

```diff
@@ -18,8 +18,8 @@ afterAll(() => {
 });
 
 describe('AreaSplineChart', function() {
-	it('should be pass correct type to billboard.js', () => {
-		new AreaSplineChart({
+	it('should be pass correct type to billboard.js', async () => {
+		await new AreaSplineChart({
 			columns: [],
```

Comment:
We should update the tests to use the new `data` API instead of the deprecated `columns` one

Reply:
Ok